frontend/test/storybook/visual-regression/v-footer.spec.ts

```diff
@@ -17,28 +16,6 @@ const storyUrl = (
   dir: LanguageDirection
 ) => `/iframe.html?id=components-vfooter--${footerKind}${dirParam(dir)}`
 
-/**
```

Comment:
The URL allows us to set the language correctly, so we don't need this function anymore